apps/ledger-live-mobile/src/families/near/accountActions.tsx

```diff
@@ -19,6 +19,9 @@ const getMainActions = ({
   parentRoute: RouteProp<ParamListBase, ScreenName>;
 }): ActionButtonEvent[] => {
   const stakingDisabled = !canStake(account);
+  const startWithValidator =
```

Comment:
```suggestion
  const shouldStartWithValidator =
```
(opinion, take or leave)